bundles/org.eclipse.emfcloud.modelserver.emf/src/org/eclipse/emfcloud/modelserver/emf/common/SchemaController.java

```diff
@@ -21,11 +21,30 @@
    @Inject
    private ModelRepository modelRepository;
 
+   @Inject
+   private SchemaRepository schemaRepository;
+
    public void getSchema(final Context ctx, final String modeluri) {
```

Comment:
It seems to me that the 'getTypeSchema' is more or less replacing the 'getSchema' method as the 'getSchema' method does not provide any useful information. If that is the case we should discuss whether it can be removed. This definitely does not have to happen as part of this change but maybe we can create a ticket for that if you agree.

Reply:
Yes I agree to remove it. As I had changes in this area anyways, I removed this method and all its occurrences.